### PR TITLE
Fix: Android weird camera permission

### PR DESCRIPTION
**Pull request by ravirajn22**
Should fix https://github.com/react-native-image-picker/react-native-image-picker/issues/1474

Need to release a new version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ If you are allowing user to capture video add `NSCameraUsageDescription` add `NS
 
 No permissions required (`saveToPhotos` requires permission [check](#note-on-file-storage)).
 
+Note: This library does not require Manifest.permission.CAMERA, if your app declares as using this permission in manifest then you have to obtain the permission before using `launchCamera`.
```
Comment (Johan-dutoit) [inline README.md:33]: Consider adding more emphasis about the whole library not requiring additional permissions.
As this comes up a lot in the issues.

File: android/src/main/java/com/imagepicker/ImagePickerModule.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import android.net.Uri;
 import android.os.Build;
 import android.provider.MediaStore;
+import android.util.Log;
```
Comment (Johan-dutoit) [inline android/src/main/java/com/imagepicker/ImagePickerModule.java:8]: Guessing this isn't needed?

**Review by Johan-dutoit — CHANGES_REQUESTED** [review]

**Comment by Johan-dutoit** [comment]
Apart from the comments, looking good!

File: README.md (hunk 1 above)
Comment (ravirajn22) [reply README.md:33]: Above this line I have mentioned `No permissions required`. So I am out of ideas.

Comment (ravirajn22) [reply README.md:33]: Lets see how the issues come then we will update appropriately. When in doubt better to avoid adding more text.

**Comment by ravirajn22** [comment]
Can we merge and release a version?

**Pull request merged**